Cargo.toml

```diff
@@ -81,7 +81,7 @@ itertools = "0.8"
 hyper = { version = "0.12", default-features = false, features = ["runtime"] }
 keys = { path = "components/keys" }
 txn_types = { path = "components/txn_types" }
-kvproto = { git = "https://github.com/pingcap/kvproto.git", default-features = false }
+kvproto = { git = "https://github.com/busyjay/kvproto.git", branch = "set-apply-term", default-features = false }
```

Comment:
Please add the kvproto dependency.

Reply:
I don't get it. Where do you want to add it as it's already written in the project toml file?

Reply:
I have given an Approve to the [PR](https://github.com/pingcap/kvproto/pull/547#pullrequestreview-363018340) which changes proto.

Reply:
We could merge that PR at first.

Reply:
LGTM